### DISCUSSION TITLE
Two different Slinger configs for the same Slingbox?

### DISCUSSION THREAD

**Discussion by robosmith** [GerryDazoo/Slinger#431] [Q&A] [answered]
@GerryDazoo We had 2 configs connecting Slinger to 2 different Pro-HDs ([Bob] & [Rick]) and that is mostly working. Sometimes great, but when the Internet gets busy, MxPlayer drops the stream often.

So I thought adding another config entry with a lower resolution might help keep the stream alive and added another config section called [BobLow] with everything the same as Bob, except Resolution=5 instead of 12. Also added sb3=BobLow to the list under [SLINGBOXES] 

However, I must have gotten something wrong because MxPlayer will not connect to /slingbox/BobLow (still connects to /slingbox/Bob).

Is there something else that would work? Thanks.

**Comment by pfuller88** [comment]
i do exactly as you describe and it works fine.  i have 4 different server settings in my ini file  for my box,   500low, 500med, 500high, 500max.  All work fine.  Must be you ini structure.  I have a custom name for my box, i use /home so my extension, for example is /home/500med.

**Comment by GerryDazoo** [comment, marked as answer]
Post your config file. This should work for sure. When you say it won't connect what exactly does that mean? Also post the server output (all of it).

**Reply by robosmith** [reply]
Ok, I probably didn't reboot Slinger after changing the config.ini. It DOES work.
